proposals/default-interface-methods.md

```diff
@@ -138,6 +138,11 @@ interface IB : IA
 
 Override declarations in interfaces may not be declared `sealed`.
 
+Public `virtual` function members in an interface may be overriden in a derived interface either implicitly (by using the `override` modifier in a public declaration in the derived interface) or explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
```

Comment:
> by using the `override` modifier in a public declaration in the derived interface

Should the accessibility instead match the member being overridden? We need to think more about implicit overrides, we might not want them to appear as a new member in the derived interface.